### PR TITLE
build: Responsive CSS breakpoints

### PR DESCRIPTION
**Pull request by eddysims**
## Motivations

We have custom media queries in Atlantis, however they were not working in Atlantis.

This PR helps make the responsive breakpoints (and all other postcss) work in Atlantis as well as Jobber.

## Changes

Adds postcss to Atlantis.

### Added

- Responsive Breakpoints page.

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- postcss working in atlantis

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

**Comment by eddysims** [comment]
@seanhealy you will notice I moved the webpack work from `.doczrc` to `gatsby-config.js`. This is the proper way to handle this in Docz 2 I believe

File: gatsby-node.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```
Comment (seanhealy) [inline gatsby-node.js:1]: Did this not work when built as a plugin or did you extract it for other reasons?

File: packages/design/src/ResponsiveBreakpoints.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+menu: Design
+route: /breakpoints
+---
+
+import { Button } from "@jobber/components/Button";
+
+# Responsive Breakpoints
+
+To make most components responsive, you will want to use the
+[useResizeObserver hook](/hooks/use-resize-observer) so that the component can
```
Comment (seanhealy) [inline packages/design/src/ResponsiveBreakpoints.mdx:12]: I wonder if we should make this a bit more prominent some how? As it is an important point that is quite different from how we are used to doing responsiveness at Jobber.

File: gatsby-node.js (hunk 1 above)
Comment (eddysims) [reply gatsby-node.js:1]: This is the proper way to handle modifying the webpack stuff in Docz 2 I believe. Im open to moving it back in, but I do like the idea of keeping the `.doczrc` a little bit cleaner.

File: packages/design/src/ResponsiveBreakpoints.mdx (hunk 2 above)
Comment (eddysims) [reply packages/design/src/ResponsiveBreakpoints.mdx:12]: @seanhealy I had thought of this in another PR, but pulled it in here. I updated the atlantis menu to bring design and hooks up to the top of the menu to make them more prominent. I think that will help users get their eyes on it. 

Thoughts?

**Comment by eddysims** [comment]
@seanhealy back to you

**Pull request merged**